File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/MyLocation.java
package com.android.blawniczak.astroweather.domain;

import android.support.annotation.NonNull;

public class MyLocation {

    private String latitude;
    private String longitude;
    private String city;
    private String woeid;

    public String getLatitude() {
        return latitude;
    }

    public String getLongitude() {
        return longitude;
    }

    public String getCity() {
        return city;
    }

    public String getWoeid() {
        return woeid;
    }

    public MyLocation(String latitude, String longitude, String woeid, String city) {
        this.latitude = latitude;
        this.longitude = longitude;
        this.city = city;
        this.woeid = woeid;
    }

    public MyLocation(String latitude, String longitude, String woeid) {
        this.latitude = latitude;
        this.longitude = longitude;
        this.city = "";
        this.woeid = woeid;
    }

    @NonNull
    @Override
    public String toString() {
        return this.latitude + " " + this.longitude  + " " + this.woeid + " " + this.city;
    }
}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/MyLocation.kt
package com.android.blawniczak.astroweather.domain

class MyLocation {

    var latitude: String? = null
        private set
    var longitude: String? = null
        private set
    var city: String? = null
        private set
    var woeid: String? = null
        private set

    constructor(latitude: String?, longitude: String?, woeid: String?, city: String?) {
        this.latitude = latitude
        this.longitude = longitude
        this.city = city
        this.woeid = woeid
    }

    constructor(latitude: String?, longitude: String?, woeid: String?) {
        this.latitude = latitude
        this.longitude = longitude
        this.city = ""
        this.woeid = woeid
    }

    override fun toString(): String {
        return this.latitude + " " + this.longitude + " " + this.woeid + " " + this.city
    }
}
